content/en/tracing/trace_collection/automatic_instrumentation/single-step-apm/troubleshooting.md

```diff
@@ -10,19 +10,157 @@ further_reading:
 
 Single Step Instrumentation (SSI) allows you to instrument applications at the host or Kubernetes level without modifying application dependencies or images. If you encounter problems enabling APM with SSI, use this guide to troubleshoot and resolve common issues. Contact [Datadog Support][1] if you need further assistance. 
 
-## Host and Docker instrumentation
+## Troubleshooting methods
+
+### Troubleshoot injection in Datadog
+
+{{< callout url="https://google.com" d_toggle="modal" d_target="#signupModal" custom_class="sign-up-trigger" btn_hidden="true" header="Join the Preview!">}}
+This feature is in preview for:
+<ul>
+<li>Languages: Python, Java, Node.js, Ruby, PHP, .NET</li>
+<li>Environments: Linux hosts, containers, Kubernetes</li>
+<li>Datadog Agent v7.68.2+</li>
+</ul>
+<p>To enable it, contact support or turn on the <code>apm-ssi-troubleshooting</code> feature flag.<p>
+{{< /callout >}}
+
+Using Datadog, you can identify and troubleshoot SSI issues across your infrastructure. You can see information like:
```

Comment:
Will customers find our usage of "SSI" here confusing? 

In general, I would avoid technical terms and use more descriptive language to avoid confusion:

```suggestion
Using Datadog, you can identify and troubleshoot instrumentation issues across your infrastructure. You can see information like:
```